app/Exceptions/Handler.php

```diff
@@ -62,6 +62,13 @@ protected function convertValidationExceptionToResponse(ValidationException $e,
             return $e->response;
         }
 
-        return response()->json($e->validator->errors()->getMessages(), 422);
+        $result ='';
```

Comment:
cemu ovo sluzi?

Reply:
Kada padne validacija, da izvuce samo poruke :) posto trenutno vraca,pr: 
[
   "password" : [ "Password must be more than 6 char" , "Passwords didn't match"],
   "user_id": ["User id is required"]
]